donkeycar/templates/complete.py

```diff
@@ -186,6 +186,11 @@ def drive(cfg, model_path=None, use_joystick=False, model_type=None,
             
         V.add(cam, inputs=inputs, outputs=outputs, threaded=threaded)
 
+    if cfg.INFO_OVERLAY:
+        from donkeycar.parts.info_overlay import InfoOverlayLogger, InfoOverlayWritter
+        V.add(InfoOverlayLogger(), inputs=['fps/current', 'user/mode', 'user/throttle', 'user/angle', 'pilot/throttle', 'pilot/angle'], outputs=['info/info_list'])
+        V.add(InfoOverlayWritter(cfg.IMAGE_W, cfg.IMAGE_H), inputs=['cam/image_array'], outputs=['cam/image_array'])
```

Comment:
You can dispatch the different images to different operations. So the original image goes into the tub and the overlay image goes into the web server or mobile app, i.e. your output here becomes `'overlay/image_array'` and you use that accordingly.